index.js

```diff
@@ -0,0 +1,151 @@
+    const nowArr = nowStr.split(" ");
+    const toDay = nowArr[0];
+    const toMonth = nowArr[1];
+    const toDate = nowArr[2];
+    const toYear = nowArr[3];
+
+    const today1 = document.createElement("h2");
+    today1.id = "day";
+    const today2 = document.createElement("h2");
+    today2.id = "date";
+    const today3 = document.createElement("h3");
+    today3.id = "YnM";
+
+    today1.innerText = toDay.toUpperCase();
+    today2.innerText = (toDate[0] === "0") ? toDate[1] : toDate;
+    today3.innerText = `${toMonth} ${toYear}`
+    today.appendChild(today1);
+    today.appendChild(today2);
+    today.appendChild(today3);
+
+    paintCalendar(y, m);
+}
+
+function paintCalendar(year, month) {
+    const days = ["SUN", "MON","TUE","WED","THU","FRI","SAT"];
+    days.forEach(function (i){
+        const addDiv = document.createElement("div");
+        const addDiv2 = document.createElement("div")        
+        addDiv.classList.add("days");
+        addDiv2.classList.add("days")
+        addDiv2.id = i;
+        const day = document.createElement("h4");
+        day.innerText = i;
+        addDiv.appendChild(day);
+        calendar.appendChild(addDiv);
+        calendar2.appendChild(addDiv2);
+    })
+
+    const lastDay = getLastday(year, month);
+
+    const firstDate = new Date(year, month, 1);
+    const firstDay = firstDate.getDay();
+
+    if (firstDay !== 0) {
+        for (var i = 0; i < firstDay; i++) {
+            var space = calendar2.querySelector(`#${days[i]}`);
+            var blank = document.createElement("br");
+            space.appendChild(blank);
+        }
+    }
+
+    for (var i = 1; i <= lastDay; i++) {        
+        var day = new Date(year, month, i);
+        var start = calendar2.querySelector(`#${days[day.getDay()]}`);
+        var date = document.createElement("button");
+        date.addEventListener("click", paintClicked);
+        date.innerText = day.getDate();
+        start.appendChild(date);
+    }
+}
+
+function paintClicked() {
+    const dd = (this.innerText !== undefined) ? this.innerText: d;
+    const clickedDate = new Date(y, m, dd);
+    const Str = clickedDate.toDateString();
+    const Arr = Str.split(" ");
+    const toDay = Arr[0];
+    const toMonth = Arr[1];
+    const toDate = Arr[2];
+    const toYear = Arr[3];
+
+    const today1 = today.querySelector("#day");
+    const today2 = today.querySelector("#date");
+    const today3 = today.querySelector("#YnM");
+
+    today1.innerText = toDay.toUpperCase();
+    today2.innerText = (toDate[0] === "0") ? toDate[1] : toDate;
+    today3.innerText = `${toMonth} ${toYear}`
+}
+
+function clearCalendar() {
+        while (calendar.firstChild !== null){
+        calendar.removeChild(calendar.firstChild);
+    }
+    while (calendar2.firstChild !== null){
+        calendar2.removeChild(calendar2.firstChild);
+    }   
+    while (today.firstChild !== null) {
+        today.removeChild(today.firstChild);
+    }
+}
```

Comment:
인덴팅과 띄어쓰기에 조금 더 신경써주세요. 그리고 위에서 코멘트한 것과 마찬가지로 !== null 표현은 아래와 같이 표현할 수 있습니다.
```suggestion
function clearCalendar() {
    while (calendar.firstChild) {
        calendar.removeChild(calendar.firstChild);
    }
    while (calendar2.firstChild) {
        calendar2.removeChild(calendar2.firstChild);
    }   
    while (today.firstChild) {
        today.removeChild(today.firstChild);
    }
}
```